cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -1191,14 +1191,23 @@ impl Inst {
                 rm,
                 ra,
             } => {
+                let mut da_size = size;
```

Comment:
small style nit, but I prefer a more functional style here, where instead of a `mut` binding, we do `let (op, size) = match alu_op { ... }` and produce original `size` or an overridden one in each match arm.